.travis.yml

```diff
@@ -36,7 +36,7 @@ install: # Use this to install any prerequisites or dependencies necessary to ru
   - cd ~/ros/ws_moveit/src
   - wstool init .
   # Download non-debian stuff
-  - wstool merge https://raw.github.com/ros-planning/moveit_docs/hydro-devel/moveit.rosinstall
+  - wstool merge https://raw.github.com/ros-planning/moveit_docs/jade-devel/moveit.rosinstall
```

Comment:
Does using `jade-devel` hold for other distros as well? I'm concerned especially because of https://github.com/ros-planning/moveit_ros/issues/670